samples/deployment/vmware/deploy.yml

```diff
@@ -70,16 +74,44 @@
           # - mgr
           # - regkey
           # - regNAT
+          fqdn: "{{ new_vm_name }}"
+          ipv4.how: "{{ vm_ipv4_how }}"
+          ipv4.addr: "{{ vm_ipv4_addr }}"
+          ipv4.mask: "{{ vm_ipv4_mask }}"
+          ipv4.gw: "{{ vm_ipv4_gw }}"
           pw: "{{ new_vm_passwd }}"
-        wait_for_ip_address: yes
-      delegate_to: localhost
-    - name: "Destroy vFTD VM"
-      vmware_guest:
-        hostname: "{{ vcenter_hostname }}"
-        username: "{{ vcenter_username }}"
-        password: "{{ vcenter_password }}"
-        datacenter: "{{ vcenter_datacenter }}"
-        name: "{{ new_vm_name }}"
-        force: yes
-        state: absent
       delegate_to: localhost
+      with_sequence: start=1 end="{{total_vms}}" format=%02x
+      register: deployed_vm
+
+    - name: Add vFTD to inventory
+      add_host:
+        name: '{{ item.invocation.module_args.name }}'
+        groups:
+          - vftd
+        ansible_host: '{{ item.invocation.module_args.properties["ipv4.addr"] }}'
+        ansible_user: admin
+        ansible_password: '{{ new_vm_passwd }}'
+        ansible_httpapi_port: 443
+        ansible_port: 443
+        ansible_httpapi_use_ssl: True
+        ansible_httpapi_validate_certs: False
```

Comment:
should we recommend to our users skip certs validation?

Reply:
Done